exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -111,7 +111,14 @@ drill.exec: {
     enabled: true,
     ssl_enabled: false,
     port: 8047
-    session_max_idle_secs: 3600 # Default value 1hr
+    session_max_idle_secs: 3600, # Default value 1hr
+    cors: {
+      enabled: true,
```

Comment:
I would default cors.enabled to false and/or set the access-cotrol-allow-origin to null. Ideally, only the end user should be able to enable CORS for all sites. 
Otherwise looks good to me.